pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
   url_launcher : ^6.0.2
   path_provider: ^2.0.1
   rxdart: ^0.26.0
+  rflutter_alert: ^2.0.4
```

Comment:
Why we are using this dependency? In anything additional it provides which is not there in material widgets ?

Reply:
This library gives us the ability to show pop up when deleting a _label_ or a _project_. It would be too cumbersome otherwise.